### PR TITLE
Control read aloud from notification

### LINKED ISSUE #621: Control read aloud from notification
#### Feature Description
Create a notification for controlling the read-aloud feature like play, pause and stop.

### PR DESCRIPTION
**Pull request by MohitMaliFtechiz**
Fixes #621

* I have added a notification with all controls (like pause, play, stop)  as mentioned in #621 .
* I have created (CoreServiceComponent, CoreServiceModule, CoreServiceScope) files for ```Injections``` like we have in app module (becasue we can not access app module in core. we can only access core module to app and custom module). this is necessary for this notification service. because we are accessing it into ```CoreReaderFragment``` which is part of ```core module``` (for accessing it into app and custom module) .
* I have not removed any control from ```ReaderFragment``` yet (play, pause, stop button in ```ReaderFragment```). it is by default like earlier.

https://user-images.githubusercontent.com/34593983/215765828-28688d81-ce0d-46a8-88d7-9f0013a4be64.mp4

### CODE REVIEW

**Comment by kelson42** [comment]
@gouri-panda  We really need a review here

**Comment by gouri-panda** [comment]
@kelson42 Ok 👍

**Review by gouri-panda — DISMISSED** [review]
@MohitMaliDeveloper LGTM! It works as expected. Works well with physical devices and emulators.

File: core/src/main/java/org/kiwix/kiwixmobile/core/read_aloud/ReadAloudService.kt [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+  private val serviceBinder: IBinder = ReadAloudBinder(this)
+  private var readAloudCallbacks: ReadAloudCallbacks? = null
+
+  override fun onCreate() {
+    CoreApp.coreComponent
+      .coreServiceComponent()
+      .service(this)
+      .build()
+      .inject(this)
+    super.onCreate()
```
[the remark this answers is not in the document]
Comment (MohitMaliFtechiz) [reply core/src/main/java/org/kiwix/kiwixmobile/core/read_aloud/ReadAloudService.kt:42]: @gouri-panda , Good catch.

[the remark this answers is not in the document]
Comment (MohitMaliFtechiz) [reply core/src/main/java/org/kiwix/kiwixmobile/core/read_aloud/ReadAloudService.kt]: done

**Comment by MohitMaliFtechiz** [comment]
@gouri-panda , I have done changes as you suggested, can you please re-review this PR.

**Review by gouri-panda — APPROVED** [review]
@MohitMaliDeveloper Thanks! Waiting for the CI to pass!

**Pull request merged**